lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -226,7 +230,11 @@ Object.defineProperties(
       const routes = new Map();
       const providerConfig = this.serverless.service.provider;
       const userConfig = providerConfig.httpApi || {};
-      this.config = { routes, id: userConfig.id };
+      this.config = {
+        routes,
+        id: userConfig.id,
+        detailedMetrics: userConfig.detailedMetrics || false,
```

Comment:
Let's make default a no setting and not `false`

Reply:
Noted, Will make the change based on your answer for https://github.com/serverless/serverless/pull/8510#discussion_r524958636